src/Directory.Build.targets

```diff
@@ -20,6 +20,11 @@
     <UpToDateCheckInput Include="$(MSBuildThisFileDirectory)..\eng\Analyzers_ShippingRules.ruleset" Condition="'$(CodeAnalysisRuleSet)' == '$(MSBuildThisFileDirectory)..\build\Analyzers_NonShippingRules.ruleset'" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="2.9.6" />
```

Comment:
Please move the package reference to eng\Versions.props

Reply:
Sorry, I meant version of the referenced package must be defined in Versions.props

Reply:
@mavasani I'll go ahead and fix this and the other issues that arose during build

Reply:
Thanks!

Reply:
Thanks @sharwell for picking this up while I slept :)